<?xml version="1.0" encoding="UTF-8"?>
<robot name="white_plate">
  <link name="plate">
    <inertial>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <mass value="1.0"/>
       <inertia ixx="0.0833" ixy="0.0" ixz="0.0" iyy="0.0833" iyz="0.0" izz="0.1667"/>
    </inertial>
    <visual>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>
        <box size="0.27 0.27 0.001" />
      </geometry>
      <material name="red">
        <color rgba="1 0.2 0.2 1"/>
      </material>
    </visual>
    <collision>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>
        <box size="0.27 0.27 0.001" />
      </geometry>
    </collision>
  </link>
  <gazebo reference="plate">
    <material>Gazebo/White</material>
  </gazebo>
</robot>
<!-- 
<?xml version="1.0" encoding="UTF-8"?>

<robot name="white_sheet">
    <link name="sheet_link">
        <inertial>
            <mass value="1.0" />
            <inertia ixx="0.001" ixy="0.0" ixz="0.0" iyy="0.001" iyz="0.0" izz="0.001" />
        </inertial>
        <visual>
            <geometry>
                <box size="0.2 0.2 0.01" />
            </geometry>
            <material>
                <color rgba="1 1 1 1" />
            </material>
        </visual>
        <collision>
            <geometry>
                <box size="0.2 0.2 0.01" />
            </geometry>
        </collision>
    </link>

    

</robot> -->